flink-end-to-end-tests/test-scripts/test_resume_savepoint.sh

```diff
@@ -47,7 +47,7 @@ ORIGINAL_DOP=$1
 NEW_DOP=$2
 STATE_BACKEND_TYPE=${3:-file}
 STATE_BACKEND_FILE_ASYNC=${4:-true}
-STATE_BACKEND_ROCKS_TIMER_SERVICE_TYPE=${5:-heap}
+STATE_BACKEND_ROCKS_TIMER_SERVICE_TYPE=${5:-rocks}
```

Comment:
Changing the default here to rocks makes sense, while I'm not sure the impact to e2e test (many cases will use rocksdb timer instead). Could you also include e2e tests in travis run to make sure no surprise? Thanks.

Reply:
The [result](https://travis-ci.com/flink-ci/flink/builds/145155686) of the e2e test verification (#10905) looks good